tox.ini

```diff
@@ -58,6 +58,7 @@ deps =
   -r{toxinidir}/requirements.txt
   -r{toxinidir}/requirements-dev.txt
   sphinx-intl
+  jupyter
```

Comment:
Is it necessary? I succeeded to run `tox -egettext` without this line and got the following message.
```
  gettext: OK (97.34=setup[60.66]+cmd[34.39,2.28] seconds)
  congratulations :) (97.44 seconds)
```

Reply:
Thank you for the details. Let's merge them as they are.